docs/api-reference/react/deckgl.md

```diff
@@ -27,17 +27,16 @@ Like any React component, `DeckGL` can accept child components. Child components
 
 ```js
 import DeckGL from '@deck.gl/react';
-import {StaticMap} from 'react-map-gl';
+import {Map} from 'react-map-gl/maplibre';
+import 'maplibre-gl/dist/maplibre-gl.css';
 
 const App = (data) => (
   <DeckGL
     initialViewState={{longitude: -122.45, latitude: 37.78, zoom: 12}}
     controller={true}
     layers={[new ScatterplotLayer({data})]}
   >
-    <StaticMap
```

Comment:
Could be good to do an audit for this and use maplibre since it doesn't need a token.

Reply:
Good idea - Add a bullet to our v9.1 tracker task?